test/cases/ostree-raw-image.sh

```diff
@@ -367,9 +368,13 @@ ISO_FILENAME="${COMPOSE_ID}-${INSTALLER_FILENAME}"
 
 greenprint "Extracting and converting the raw image to a qcow2 file"
 sudo xz -d "${ISO_FILENAME}"
-sudo qemu-img convert -f raw "${COMPOSE_ID}-image.raw" -O qcow2 "${IMAGE_KEY}-uefi.qcow2"
-sudo mv "${IMAGE_KEY}-uefi.qcow2" /var/lib/libvirt/images/
-LIBVIRT_IMAGE_PATH=/var/lib/libvirt/images/${IMAGE_KEY}-uefi.qcow2
+sudo qemu-img convert -f raw "${COMPOSE_ID}-image.raw" -O qcow2 "${IMAGE_KEY}-raw-image.qcow2"
+
+# Prepare qcow2 disk
+BIOS_LIBVIRT_IMAGE_PATH=/var/lib/libvirt/images/${IMAGE_KEY}-bios.qcow2
+UEFI_LIBVIRT_IMAGE_PATH=/var/lib/libvirt/images/${IMAGE_KEY}-uefi.qcow2
+sudo cp "${IMAGE_KEY}-raw-image.qcow2" BIOS_LIBVIRT_IMAGE_PATH
+sudo cp "${IMAGE_KEY}-raw-image.qcow2" UEFI_LIBVIRT_IMAGE_PATH
```

Comment:
Why not just keep them as one variable, without the `-{uefi,bios}` suffix? No need to respin just for that, but thought I'd point it out :)

Reply:
Fixed, thank you!